be/sidedish-be/src/main/java/com/codesquad/sidedish/utils/SampleDataFactory.java

```diff
@@ -1,17 +1,41 @@
 package com.codesquad.sidedish.utils;
 
+import com.codesquad.sidedish.web.sidedish.ItemDTO;
 import com.codesquad.sidedish.web.sidedish.SidedishDTO;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
 import org.springframework.core.io.ClassPathResource;
 
+import java.io.File;
 import java.io.IOException;
 import java.util.List;
 
 public class SampleDataFactory {
-    public static List<SidedishDTO> createBestMenus() {
+
```

Comment:
```suggestion
    private SampleDataFactory() {
    }
```
static만 사용하면 private으로 막아주면 좋을 것같습니다